### ISSUE TITLE
Issues with the Quickstart recommendations

### ISSUE THREAD

**Issue by gsaha009** [columnflow/columnflow#271]
Hello CF developers,

We are going to start a physics analysis using Columnflow. As a starting point, we used the command 
`bash -c "$(curl -Ls https://raw.githubusercontent.com/columnflow/columnflow/master/create_analysis.sh)"` to build an example analysis structure. We then executed rest of the commands one by one and,	we encountered a few problems. Please find those in the following:

*** work area: `/afs/cern.ch/work/g/gsaha/public/IPHC/Work/HCPtoTauTau`

1. `law run cf.ReduceEvents --version dev1 --branch 0`
--> In `calibration/example.py`, it could not locate "nJet" in	@calibrator(uses={..},..)
--> After removing "nJet", it got stuck at: `for values in itertools.product(*(unique_group_values[g] for g in group_names)): KeyError: 'process'`. Then after including the `process` dictionary in the `group map`, it ran fine.
--> Some problem with JSONs, like `json_mirror`, `muon_sf` and `external_files` in `HCPtoTauTau/hcp/config/analysis_hcp.py`

2. `law run cf.PlotVariables1D --version dev1 --datasets 'st*' --variables jet1_pt`
--> Exception: `deterministic_event_seeds` did not receive any columns matching: `GenJet.pt`, `GenPart.pt`.
It runs smoothly either we comment out the line `events = self[deterministic_seeds](events, **kwargs)` in `production/example.py`, or comment the whole function `deterministic_event_seeds_init` in `modules/columnflow/columnflow/pr\
oduction/cms/seeds.py` after adding `optional("GenJet.pt"),optional("GenPart.pt")` in `deterministic_event_seeds`.

Finally, we managed to produce the plots after we did the changes mentioned above, but we are not entirely certain if everything is woking fine under the hood. On a different note, this issue was not present in an older commit [bc26321f138dbc700259fdffc5a0318149b4d50a], but the plots were empty that time. Please have a look at the changes we had to make and let us know your comments.
Thanks.

Gourab
(on behalf of the Higgs CP tau tau analysis team, IPHC)

**Comment by riga** [comment]
Dear @gsaha009 ,

Thanks for the detailed report! I'm looking into this right away and get back to you.

Side note: We were aware that the first external users were going to start using columnflow soon, which is why we did some internal changes that we had on our list for a while. Some of these changes were added last and this week, with a couple things still left open, but they're going to be added soon. Therefore, issues such as those you experienced can happen during the next weeks, but they should - of course - be fixed in the medium / long term.

Welcome aboard ;)

**Comment by gsaha009** [comment]
Hi @riga 
Thanks for your quick response :) 
Yeah, I noticed many commits in the last couple of weeks. 
Btw, at this moment do you guys have any plan to develop the documentation? 
Thanks :)

**Comment by riga** [comment]
Yes, as a matter of fact, we had a meeting with a couple developers today on how to push the docs as fast as possible, so there will be some updates soon too.

**Comment by riga** [comment]
Hi @gsaha009 ,

as expected, the issues you saw were related to some of the latest changes and I fixed them in 5265cba.

> Some problem with JSONs, like json_mirror, muon_sf and external_files in HCPtoTauTau/hcp/config/analysis_hcp.py

I mistakenly deleted a mirror for the json pog files in my personal public afs space but I recovered it, so this also should not be an issue anymore. We (CAT) will get in touch XPOG to make sure that the centrally provided json pog files are available on a commit bash basis (same as I did in my personal space).
